ReactSkia/components/RSkComponentText.cpp

```diff
@@ -37,9 +37,9 @@ void RSkComponentParagraph::OnPaint(
       *std::static_pointer_cast<ParagraphProps const>(shadowView.props);
   auto data = state->getData();
   auto text = data.attributedString.getString();
-  auto fontSize = props.textAttributes.fontSize;
-  float ratio = 255.9999;
-  auto color = colorComponentsFromColor(props.textAttributes.foregroundColor);
+  auto fontSize = !std::isnan(props.textAttributes.fontSize) ? props.textAttributes.fontSize : TextAttributes::defaultTextAttributes().fontSize;
+  float ratio = 256.0;
```

Comment:
Thanks for revising the ratio.
Could you please also help me to revise the ratio in `RSkComponentView.cpp` ?

Reply:
Using 255.9999 was actually correct since biggest color component value is 255. For example, for the alpha value of 1.0,  color.alpha * ratio = 1.0 x 255.9999  = 255.9999.  When this value is passed to SkColorSetARGB(), it will be casted to int and will become 255. 

I have now updated the PR with the original value of 255.9999.

Reply:
Thank you @ekegulskiy for clarification. I am going to merge this.